Sparkle/SUTouchBar.m

```diff
@@ -0,0 +1,94 @@
+//
+//  SUTouchBar.m
+//  Sparkle
+//
+//  Created by Yuxin Wang on 05/01/2017.
+//  Copyright © 2017 Sparkle Project. All rights reserved.
+//
+
+#import "SUTouchBar.h"
+
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101202
```

Comment:
I'm not sure if that will work on older systems. This checks for the SDK version, rather than the OS version Sparkle is running on.